website/docs/guides/delivery.mdx

```diff
@@ -0,0 +1,473 @@
+
+This guide uses Flux manifests to install Flagger and Linkerd. Using Flux allows us to manage our cluster applications in a declarative way through changes in a Git repository.
+
+## Installing Flagger using Flux
+
+For the Flagger installation, a Kustomization file will be used.
+
+First, add a Namespace resource for Flagger:
+
+```yaml title="flagger/namespace.yaml"
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: flagger
+```
+
+Then, to make the Flagger Helm repository available in the cluster, add the following manifest to the cluster repository:
+
+```yaml title="flagger/source.yaml"
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: flagger
+spec:
+  interval: 1h
+  url: https://flagger.app
+```
+
+Then, to install the latest version of Flagger and the load tester app (which is used to generate traffic during the analysis phase), add the following `HelmRelease` manifests to the cluster repository:
+
+```yaml title="flagger/releases.yaml"
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: flagger
+spec:
+  releaseName: flagger
+  install:
+    crds: Create
+  upgrade:
+    crds: CreateReplace
+  interval: 10m
+  chart:
+    spec:
+      chart: flagger
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: flagger
+  values:
+    metricsServer: http://prometheus.linkerd-viz:9090
+    meshProvider: linkerd
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: loadtester
+spec:
+  interval: 10m
+  chart:
+    spec:
+      chart: loadtester
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: flagger
+```
+
+In this case, Flagger is configured to work with the Linkerd service mesh. [Other providers](https://docs.flagger.app/install/flagger-install-on-kubernetes) may also be used instead of Linkerd to integrate with Flagger.
+
+Finally,  add the following Kustomization file that references all the previous files that were added:
+
+```yaml title="flagger/kustomization.yaml"
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: flagger
+resources:
+- namespace.yaml
+- source.yaml
+- releases.yaml
+```
+
+## Installing Linkerd using Flux
+
+For the Linkerd installation, a Kustomization file will be used.
+
+In order to support mTLS connections between meshed pods, Linkerd requires a trust anchor certificate and an issuer certificate with its corresponding key. These certificates are automatically created when the `linkerd install` command is used but when using a Helm chart to install Linkerd, these certificates need to be provided.
+
+Follow [these instructions](https://linkerd.io/2.11/tasks/generate-certificates/#generating-the-certificates-with-step) to generate the certificates using the `step` CLI.
```

Comment:
I may be wrong... but i think you're summarising the steps below, so the reader doesnt have to go to this link right?

Reply:
That's a good point, I'll remove these.